src/AppBundle/Resources/views/Ndr/IncomeBenefit/_answers.html.twig

```diff
@@ -30,170 +32,194 @@
                     {% endfor %}
                 </ul>
             {% endif %}
-        </td>
-        <td class="change-answer">
-            <a href="{{ path('ndr_income_benefits_step', { 'ndrId': ndr.id, 'step': 1, 'from': 'summary' }) }}" class="behat-link-edit">{{ 'edit' | trans({}, 'common') }}</a>
-        </td>
-    </tr>
+        </dd>
+        <dd class="govuk-summary-list__actions">
+            <a class="govuk-link behat-link-edit" href="{{ path('ndr_income_benefits_step', { 'ndrId': ndr.id, 'step': 1, 'from': 'summary' }) }}">
+                {{ 'edit' | trans({}, 'common') }}
+            </a>
+        </dd>
+    </div>
+
     {% set stateBenefitsOther = ndr.getStateBenefitOther() %}
     {% if stateBenefitsOther.present %}
-        <tr class="behat-region-benefits-other-more-details">
-            <td>
+        <div class="govuk-summary-list__row">
```

Comment:
Suspect this needs `behat-region-benefits-other-more-details` class for tests to pass